spatialos-sdk/src/worker/component.rs

```diff
@@ -53,6 +53,60 @@ where
     fn get_response_command_index(response: &Self::CommandResponse) -> u32;
 }
 
+/// Additional parameters for sending component updates.
+///
+/// Additional parameters passed to [`Connection::send_component_update`]. Note that
+/// all parameters are kept private and the struct can only be initialized with
+/// default values in order to make it possible to add new parameters without a
+/// breaking change.
+///
+/// If you would like to use a method-chaining style when initializing the parameters,
+/// the [tap] crate is recommended. The examples below demonstrate this.
+///
+/// # Parameters
+///
+/// * `loopback: false` allow the update to be sent back to the worker without waiting
+///   to be routed through SpatialOS. This allows the worker to receive the update op
+///   faster, but risks that the worker won't know if the update actually failed. [See
```

Comment:
Technically, the worker doesn't know if the update failed either way! The difference is you could hit a race condition with authority changing mid tick I think

Reply:
Ah, good point. I'll update the wording to more accurately reflect the caveats with short-circuiting.